### PR TITLE
Add tooltip accessibility doc

### LINKED ISSUE #4068: [Accessibility Docs] Tooltips
Using [this template](https://docs.google.com/document/d/1IwFMfJrTfJPXDcVJfLWXYkw-pjgWhBWo4GX8X098Q64/edit#), create accessibility docs for [Tooltips](https://vanillframework.io/docs/patterns/tooltips)

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Add accessibility notes for tooltips
- Filed an [issue](https://github.com/canonical-web-and-design/vanilla-framework/issues/4240) as on our example currently the description is read out twice

Fixes #4068 

## QA

- Open [demo](insert-demo-url)
- Check the tooltips accessibility section

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

- [x] PR should have one of the following labels to automatically categorise it in release notes:
  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
- [x] Vanilla version in `package.json` should be updated relative to the [most recent release](https://github.com/canonical-web-and-design/vanilla-framework/releases/latest), following semver convention:
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
- [x] Any changes to component class names (new patterns, variants, removed or added features) should be listed on the [what's new page](https://github.com/canonical-web-and-design/vanilla-framework/blob/main/templates/docs/whats-new.md).
- [x] Documentation side navigation should be updated with the relevant labels.

### CODE REVIEW

File: templates/docs/patterns/tooltips.md [hunk 1]
```diff
@@ -36,6 +36,31 @@ In some cases you may need the tooltip element to exist outside of the element i
...
+### How it works
+
+A tooltip is text that appears in a small overlay on demand, usually when hovering over the thing it describes. It is hidden by default, and becomes available on hover or focus of the control it describes. It should provide information that isn’t self explanatory or well known.
+
+### Considerations
+
+This component strives to follow [WCAG 2.1 (level AA) guidelines](https://www.w3.org/TR/WCAG21/), and care must be taken to ensure this effort is maintained when the component is implemented across other projects. This section offers advice to that effect:
+
+- Be sure to include the `aria-describedby` attribute with the `id` of the description.
+- They should be discoverable, not placed over plain text.
```
Comment (bartaz) [inline templates/docs/patterns/tooltips.md:52]: Not sure, what "not placed over plain text" means here?

**Review by bartaz — APPROVED** [review]
Small comments and question, but overall LGTM.

File: templates/docs/patterns/tooltips.md (hunk 1 above)
Comment (bethcollins92) [reply templates/docs/patterns/tooltips.md:52]: I think it means don't use something which isn't obviously a tooltip, like a word in the middle of a paragraph. I'll reword this to make it more clear
